UPGRADE.md

```diff
@@ -27,12 +27,10 @@ Follow instructions in a [monorepo upgrade guide](docs/contributing/upgrading-mo
     * *(MacOS, Windows only)* run `docker-sync stop`
     * *(MacOS, Windows only)* run `docker-sync clean` so your volumes will be removed
     * follow upgrade notes in a *Infrastructure* section (related with `docker-compose.yml`, `Dockerfile`, docker containers, `nginx.conf`, `php.ini`, etc.)
-    * change all the microservices image versions in your `docker-compose.yml` to version you are upgrading to
-        eg. `image: shopsys/microservice-product-search:v7.0.0-beta1`
     * *(MacOS, Windows only)* run `docker-sync start` to create volumes  
-    * run `docker-compose up -d --build --force-recreate` to start application again
+    * run `docker-compose up -d --build --force-recreate --remove-orphans` to start application again
```

Comment:
is it really necessary to add the --remove-orphans? Is it because of removal of microservices?

Reply:
yes